miniupnpd/commonrdr.h

```diff
@@ -64,4 +64,20 @@ update_portmapping_desc_timestamp(const char * ifname,
                    unsigned short eport, int proto,
                    const char * desc, unsigned int timestamp);
 
+#ifdef USE_NFTABLES
+/* only provided by nftables implementation at the moment */
+
+typedef enum {
+	TABLE_NAME,
+	TABLE4_NAME,
+	TABLE6_NAME,
+	NAT_CHAIN_NAME,
+	NAT_POSTROUTING_CHAIN_NAME,
+	FORWARD_CHAIN_NAME,
+} rdr_name_type;
```

Comment:
shouldn't you use more specific names like NF_TABLE_NAME ?

Reply:
Good point - enums are scoped to the class that contains them in C++, but they have file scope in C - I'd forgotten that. I'll make them more unique.